kinto/__main__.py

```diff
@@ -30,6 +30,8 @@ def main(args=None):
                         dest='backend',
                         required=False,
                         default=None)
+    parser.add_argument('--version', action='version', version=__version__,
```

Comment:
I believe this will do it by default. In which case it could be good to define it explicitely here, you're right.